Modules/Core/Common/include/itkThreadLogger.h

```diff
@@ -59,14 +59,7 @@ class ITKCommon_EXPORT ThreadLogger : public Logger
   using DelayType = unsigned int;
 
   /** Definition of types of operations for ThreadLogger. */
-  typedef enum
-  {
-    SET_PRIORITY_LEVEL,
-    SET_LEVEL_FOR_FLUSHING,
-    ADD_LOG_OUTPUT,
-    WRITE,
-    FLUSH
-  } OperationType;
+  using OperationType = enum { SET_PRIORITY_LEVEL, SET_LEVEL_FOR_FLUSHING, ADD_LOG_OUTPUT, WRITE, FLUSH };
```

Comment:
The proposed `using` appears unnecessary here, just like the original `typedef` was. I would prefer to simply write:

     enum OperationType { SET_PRIORITY_LEVEL, SET_LEVEL_FOR_FLUSHING, ADD_LOG_OUTPUT, WRITE, FLUSH };